### PR TITLE
feat: make withdrawal denomination more obvious

### PR DESCRIPTION
**Pull request by patcito**
Here it what it looks like if denomination is the same:
samedenom

and if it's different:
diffdenom

### CODE REVIEW

**Comment by patcito** [comment]
latest

localhost_3000_vaults_0x2f08119C6f07c006695E079AAFc638b8789FAf18

**Comment by patcito** [comment]
latest

image

File: app/components/VaultControls/index.js [hunk 1]
```diff
@@ -1086,6 +1124,24 @@ export default function VaultControls(props) {
...
+                      >
+                        <path
+                          fillRule="evenodd"
+                          clipRule="evenodd"
+                          d="M19.7593 12.6508C20.0802 12.2763 20.0802 11.7237 19.7593 11.3492L13.7593 4.34923C13.3998 3.9299 12.7685 3.88134 12.3492 4.24076C11.9299 4.60018 11.8813 5.23148 12.2407 5.65081L17.6829 12L12.2407 18.3492C11.8813 18.7686 11.9299 19.3999 12.3492 19.7593C12.7685 20.1187 13.3998 20.0701 13.7593 19.6508L19.7593 12.6508ZM6.75926 19.6508L12.7593 12.6508C13.0802 12.2763 13.0802 11.7237 12.7593 11.3492L6.75926 4.34923C6.39983 3.9299 5.76853 3.88134 5.34921 4.24076C4.92988 4.60018 4.88132 5.23148 5.24074 5.65081L10.6829 12L5.24074 18.3492C4.88132 18.7686 4.92988 19.3999 5.34921 19.7593C5.76853 20.1187 6.39983 20.0701 6.75926 19.6508Z"
+                          fill="white"
+                        />
+                      </svg>
+                    </Box>
+                  ) : null}
```
Comment (nymmrx) [inline app/components/VaultControls/index.js:1144]: Could we abstract this into a component (e.g. DoubleArrowRightIcon) so we don't add many extra lines to this super large file?

**Review by nymmrx — CHANGES_REQUESTED** [review]
Minor code layout change request, everything else looks good to me on the logical side.

Graphically speaking i don't really love the amount of  blank space to the right of the controls. We could move the text to the right instead of on top, to fill that space.

File: app/components/VaultControls/index.js (hunk 1 above)
Comment (patcito) [reply app/components/VaultControls/index.js]: ok, pushed

**Review by x48-crypto — APPROVED** [review]

**Comment by x48-crypto** [comment]
@nymmrx is this good to merge?

**Review by nymmrx — APPROVED** [review]
lgtm

**Pull request merged**